main.c

```diff
@@ -117,8 +117,13 @@ static bool is_in_usable_coreboot_memory_region(const struct memory_region regio
     return false;
 }
 
-// Linux boot follows the Linux x86 32-bit Boot Protocol
+// Boots the provided payload according to the Linux x86 32-bit Boot Protocol
 // (https://www.kernel.org/doc/html/latest/x86/boot.html#bit-boot-protocol).
+//
+// Only bzImage Linux kernels are supported.
+//
+// We require Linux boot protocol 2.10 and above. We always satisfy the preferred load
+// address of the kernel, even if it is relocatable.
```

Comment:
No. We only support relocatable kernels currently and leave it where `fw_cfg` tells us to find it. The loader is not relocating.

Reply:
good catch! thanks.